.travis.yml

```diff
@@ -5,18 +5,32 @@ python:
   - "3.4"
 
 addons:
-  apt:
-    packages:
-      - oracle-java8-set-default
```

Comment:
Was this all that was needed to fix the JDK? I think it broke when Travis moved everyone to Xenial by default.

Reply:
This rings a bell. I think i had to do this in the api as well.

Reply:
Yep, I think I copied it from Christian's PR

Reply:
yep